src/modules/nav/nav.component.scss

```diff
@@ -6,8 +6,12 @@
   flex-direction: column;
 }
 
+.stache-nav-list .stache-nav-list {
```

Comment:
Is this `padding-left` intended for sidebar navigations, though? For example, if we threw a child nav on the breadcrumbs, it would add padding to the left-hand side. `NavComponent` is designed to be bare-bones. If it makes more sense, we can use the `.stache-nav-sidebar` classes instead...